public/index.html

```diff
@@ -40,7 +30,7 @@
     <title>Darwinia Network - 达尔文网络</title>
     <meta name="keywords" content="达尔文网络 darwinia 区块链 智能合约 token ethereum blockchain" />
```

Comment:
```suggestion
    <meta name="keywords" content="Darwinia Smart Contract Cross-chain Message Blockchain Substrate Polkadot" />
```

Reply:
I just made a commit of your suggestions